lib/engine_mustache.js

```diff
@@ -19,12 +19,13 @@
 
 "use strict";
 
-var Mustache = require('mustache');
+var Hogan = require('hogan.js');
+var JSON5 = require('json5');
 var utilMustache = require('./util_mustache');
 
 var engine_mustache = {
-  engine: Mustache,
-  engineName: 'mustache',
+  engine: Hogan,
+  engineName: 'hogan',
```

Comment:
@bmuenzenmeyer, @dmolsen and I are considering whether to keep the name of the engine 'mustache'. It still exists to support Mustache templates, after all -- Hogan is an implementation detail.